pkg/cmd/step/verify/step_verify_preinstall.go

```diff
@@ -679,6 +708,22 @@ func (o *StepVerifyPreInstallOptions) gatherGitRequirements(requirements *config
 			}
 		}
 	}
+	if len(requirements.Cluster.DevEnvApprovers) == 0 && !o.BatchMode {
+		approversString, err := util.PickValue(
+			"Comma-separated git provider usernames of approvers for development environment repository",
+			"",
+			true,
+			"Pull requests to the development environment repository require approval by one or more "+
+				"users, specified in the 'OWNERS' file in the repository. Please specify a comma-separated "+
+				"list of usernames for your Git provider to be used as approvers.",
+			o.GetIOFileHandles())
+		if err != nil {
+			return errors.Wrap(err, "error configuring approvers for development environment repository")
```

Comment:
nit: you don't need to prefix the error messages with `error`.  The message it will look like: `error: error configuring ...`.

Reply:
Doh, will fix.